source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/calculation/CalculationJobType.java

```diff
@@ -63,10 +65,19 @@ public enum CalculationJobType {
 
   /**
    * Only 1 Reference scenario
-   *
+   * <p>
```

Comment:
I don't really mind this, but we don't use these tags that often and it does seem inconsistent with the other javadocs in this enum.